plugins/techdocs/src/reader/components/TechDocsPageHeader.tsx

```diff
@@ -112,8 +112,8 @@ export const TechDocsPageHeader = ({
       subtitle={
         siteDescription && siteDescription !== 'None' ? siteDescription : ''
       }
-      type={name}
-      typeLink={componentLink(entityId)}
+      type="Docs"
+      typeLink="/docs"
```

Comment:
It's highly likely that this will be the correct URL for TechDocs plugin, but we should not assume this.

If you're up for it, the TechDocs API has an instance of something called [`discovery API`](https://github.com/backstage/backstage/blob/e07b41751e18c8f633d5d6430b2619d6e7def05c/plugins/techdocs/src/api.ts#L53) which can give you the base URL with `.getBaseUrl()`.

Reply:
We can find out `'/docs'` by `useRouteRef(rootRouteRef)()` where `useRouteRef` comes from `@backstage/core` and `rootRouteRef` is defined inside `plugins/techdocs/plugin.ts`